### PR TITLE
Update content on landing pages

### PR DESCRIPTION
**Pull request by dashouse**
This adds content to the Styles page and amends the Components and Patterns pages to be more in-line with the current build.

### CODE REVIEW

File: src/patterns/index.md.njk [hunk 1]
```diff
@@ -9,3 +9,8 @@ tasks and scenarios.
 Some of the patterns in this section contain coded examples, others are written
 guidance. Patterns often use single or combinations of components and describe
 their use in context of their relationship to the user.
+
```
Comment (joelanman) [inline src/patterns/index.md.njk]: Should this be 'For the most part'? Has @amyhupe looked at these?

File: src/patterns/index.md.njk [hunk 2]
```diff
@@ -9,3 +9,8 @@ tasks and scenarios.
...
+On the most part, if you are using the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com) or
```
Comment (joelanman) [inline src/patterns/index.md.njk]: Should this be 'for the most part'? Has @amyhupe looked at these?

Comment (dashouse) [reply src/patterns/index.md.njk]: @joelanman Good shout, this is with Amy now

**Comment by dashouse** [comment]
Content reviewed an updated so merging

**Pull request merged**